sys/kern/imgact_elf.c

```diff
@@ -610,13 +612,16 @@ __elfN(build_imgact_capability)(struct image_params *imgp,
 	}
 
 	/*
-	 * Note: vm_map_reservation_create aligns down, but we have to align
-	 * upwards here to avoid rounding down to zero for the main executable.
-	 * For RTLD we also align upwards to avoid aligning down into the
-	 * memory region for the main binary.
+	 * Note: vm_map_reservation_create aligns down, but we have to
+	 * align upwards here to avoid rounding down to zero for the main
+	 * executable. For RTLD we also align upwards to avoid aligning
+	 * down into the memory region for the main binary.
 	 */
 	reservation = roundup2(start + rbase,
```

Comment:
If you do `reservation = start + rbase` before the ifdef and do `reservation = roundup2(reservation, ...)` here then you can shave off needing an #else